jatszohaz/templates/jatszohaz/profile_detail.html

```diff
@@ -58,5 +59,25 @@ <h3>{% trans 'Rents' %}</h3>
         {% else %}
             {% trans "No rents yet." %}
         {% endif %}
+
+        <h3>{% trans 'Comments' %}</h3>
+        {% for comment in comments %}

```

Comment:
Ide hozzá lehetne adni valami kis leírást, hogy ezek a kommentek csak a körtagoknak jelennek meg és csak a körtagok tudnak újat létrehozni. Ez azért fontos, hogy egyértelmű legyen mindenkinek mi is a célja ennek és hogy működik.